jpm/jpm.d.ts

```diff
@@ -0,0 +1,1114 @@
+// Type definitions for Firefox Addon SDK
+// Project: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/Add-on_SDK
+// Definitions by: Mohammed Hamdy <https://github.com/github-account-because-they-want-it>
```

Comment:
The CI uses this to ping authors to review any changes in the definition later on.